test/core/comment.test.ts

```diff
@@ -0,0 +1,15 @@
+import { describe, it, expect } from 'bun:test';
+import { getAllComments } from '../../src/core/fetch/comment';
+import { chainId, daoAddress, apiUrl } from '../constants';
+
+describe('Comment', () => {
```

Comment:
how's [code coverage](https://bun.sh/docs/test/coverage) with all these tests? it could help us get a sense about how many code has been cover by tests.